fidesctl/src/fidesctl/core/config/cli_settings.py

```diff
@@ -11,7 +15,23 @@ class CLISettings(FidesSettings):
 
     local_mode: bool = False
     server_url: str = "http://localhost:8080"
-    analytics_id: str = generate_client_id(FIDESCTL_CLI)
+    analytics_id: Optional[str]
+
+    @validator("analytics_id", always=True)
+    def ensure_analytics_id_override(cls, value: str) -> str:
+        """
+        Resolves the `analytics_id` value in the following order of priority:
+        1. The value of the `ANALYTICS_ID_OVERRIDE` environment variable
+        2. The value of the `FIDESCTL__CLI__ANALYTICS_ID` environment variable
+        3. The value included in the config file
+        4. A newly generated value
+        """
+
+        override_id = getenv("ANALYTICS_ID_OVERRIDE")
+        if override_id is not None and override_id != "":
+            return override_id
+
+        return value if value != "" else generate_client_id(FIDESCTL_CLI)
```

Comment:
Currently if `ANALYTICS_ID_OVERRIDE` is not set by `FIDESCTL__CLI__ANALYTICS_ID`, I get the following error:
```
assert arg is not None and arg != "", (
AssertionError: arg must be provided
```